src/main/plugins/org.dita.base/xsl/preprocess/maplinkImpl.xsl

```diff
@@ -590,10 +591,21 @@ See the accompanying LICENSE file for applicable license.
           <!-- add desc node and text -->
           <xsl:apply-templates select="*[contains(@class, ' map/topicmeta ')]/*[contains(@class, ' map/shortdesc ')]"/>
         </xsl:if>
+        <xsl:apply-templates select="*[contains(@class,' ditaot-d/ditaval-endprop ')]" mode="add-props-to-link"/>
       </link>
     </xsl:if>
   </xsl:template>
   
+  <xsl:template match="@*|*|text()" mode="add-props-to-link">
```

Comment:
Could this be `@* | node()` to be a bit more generic. Same for the apply-templates expression below.

Reply:
Updated.